randovania/games/samus_returns/json_data/Area 1.txt

```diff
@@ -416,6 +416,15 @@ Extra - asset_id: collision_camera_017
           Lay Any Bomb
           Super Jump (Intermediate) or Unmorph Extend (Intermediate) or Climb Rooms Vertically (High Jump)
 
+> Pickup (DNA); Heals? False
+  * Layers: default
+  * Pickup 176; Category? Minor
+  * Extra - pickup_type: metroid
+  * Extra - item_id: ITEM_RANDO_DNA_5
```

Comment:
What's the purpose of all the `item_id` in the extra section?

Reply:
I guess nothing since it doesn't do anything. My idea is to use these custom dna pickups so they can be hinted at, and then each one grant's `ITEM_ADN` to satisfy the patcher requirement.

Reply:
Hints would come from RDV which doesnt need these kinds of item IDs. you just give it the actual pickup instead that you want the hint for.